int64 mission
string type
---
singleOdlc target_info